x/subscription/types/keys.go

```diff
@@ -17,6 +17,22 @@ const (
 	MemStoreKey = "mem_subscription"
 )
 
+const (
+	// SubscriptionKeyPrefix is the prefix to retrieve all Subscription
+	SubscriptionKeyPrefix = "Subscribe/"
+)
+
 func KeyPrefix(p string) []byte {
 	return []byte(p)
 }
+
+// SubscriptionKey returns the store key to retrieve a Subscription from the consumer field
+func SubscriptionKey(consumer string) []byte {
+	var key []byte
+
+	indexBytes := []byte(consumer)
+	key = append(key, indexBytes...)
+	key = append(key, []byte("/")...)
```

Comment:
change to "_"

Reply:
copied from x/pairing/.., x/spec/..., x/conflict/...